src/module.ts

```diff
@@ -120,6 +123,12 @@ export default defineNuxtModule({
       await installModule('@nuxt/postcss8')
     }
 
+    // Add support for @nuxt/content if module is present
+    if (!moduleOptions.disableContentSupport && (nuxt.options.modules.includes('@nuxt/content') || nuxt.options.modules.includes('@nuxt/content-edge'))) {
+      tailwindConfig.content = tailwindConfig.content ?? []
+      tailwindConfig.content.push(`${nuxt.options.buildDir}/cache/content/parsed/**/*.md`)
```

Comment:
Please avoid checking module existence like this! Instead use a hook that only activates with module and also this could be from nuxt-content to extend tailwind config

Reply:
Thanks for the suggestion @pi0 ; I've moved this part into https://github.com/nuxt/content/pull/1351 💚